src/modules/facility-activation-transaction/dto/get-facility-activation-transaction-response.dto.ts

```diff
@@ -0,0 +1,130 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ENUMS, EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers';
+
+export class GetFacilityActivationTransactionResponseDto {
+  @ApiProperty({
```

Comment:
Numerous response properties can have enum, should they be allocated.

Reply:
Which properties did you mean @abhi-markan? I think I'm currently using enums for all the properties that have possible values listed in the Anypoint Exchange docs